README.md

```
@@ -370,24 +339,24 @@ plot(eyeris_preproc,
   add_progressive_summary = TRUE
 )
-#> ℹ [2025-10-06 19:54:25] [INFO] Plotting block 1 with sampling rate 1000 Hz from
+#> ℹ [2026-01-27 17:35:36] [INFO] Plotting block 1 with sampling rate 1000 Hz from
 #> possible blocks: 1
 ```
 
-<img src="man/figures/README-timeseries-plot-1.png" width="100%" /><img src="man/figures/README-timeseries-plot-2.png" width="100%" /><img src="man/figures/README-timeseries-plot-3.png" width="100%" /><img src="man/figures/README-timeseries-plot-4.png" width="100%" /><img src="man/figures/README-timeseries-plot-5.png" width="100%" /><img src="man/figures/README-timeseries-plot-6.png" width="100%" />
+<img src="man/figures/README-timeseries-plot-1.png" alt="" width="100%" /><img src="man/figures/README-timeseries-plot-2.png" alt="" width="100%" /><img src="man/figures/README-timeseries-plot-3.png" alt="" width="100%" /><img src="man/figures/README-timeseries-plot-4.png" alt="" width="100%" /><img src="man/figures/README-timeseries-plot-5.png" alt="" width="100%" /><img src="man/figures/README-timeseries-plot-6.png" alt="" width="100%" />
 
-    #> ℹ [2025-10-06 19:54:26] [INFO] Creating progressive summary plot for block_1
+    #> ℹ [2026-01-27 17:35:37] [INFO] Creating progressive summary plot for block_1
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)